core-bundle/contao/themes/flexible/styles/basic.css

```diff
@@ -1083,3 +1074,31 @@ a.tl_submit {
 		margin-bottom: 1px;
 	}
 }
+
+/* BC */
+*:not(.widget-group) > .clr {
+	clear:both;
+	width:calc(100% - 30px);
+}
+*:not(.widget-group) > .clr:not(.widget) {
+	width:100%;
+}
+*:not(.widget-group) > .clr:before {
+	content:"";
+	display:table;
+}
+*:not(.widget-group) > .w50 {
+	width:calc(50% - 30px);
+	float:left;
+	min-height:80px;
+}
+*:not(.widget-group) > .long {
+	width:calc(100% - 30px); /* see #6320 */
+}
+*:not(.widget-group) > .m12 {
+	margin: 0 15px;
+}
```

Comment:
The coding style still needs to be adjusted here :) (space after `:` and break after `}`).

Reply:
Fixed in 87c308d23bf6a1d594a280a06b00593682253bd3